### PR TITLE
Add getRole function to User class and fix the node engine

### PR DESCRIPTION
**Pull request by JoMas971**

### CODE REVIEW

**Comment by JoMas971** [comment]
@cfsnyder i've moved the `getRole()` function to the user class since we need it to get the user role for a specific company. But i've kept the rest separate. I'll let you merge this if it's ok for you ;-).

File: src/sdk/model/user.ts [hunk 1]
```diff
@@ -224,6 +225,18 @@ export class User {
     });
   }
 
+  /**
+   * Gets the user's role for a company
+   * @param {string} companyUuid
+   * @returns {Promise<Role>}
+   */
+  async getRoleFor(companyUuid: string): Promise<Role> {
```
Comment (cfsnyder) [inline src/sdk/model/user.ts]: can we change the name to `getRole`

Comment (JoMas971) [reply src/sdk/model/user.ts]: sure. let me do that

**Comment by JoMas971** [comment]
@cfsnyder it should be good now ;-).

**Comment by cfsnyder** [comment]
It has conflicts @JoMas971

**Comment by JoMas971** [comment]
Should be good now @cfsnyder

**Pull request merged**